plugins-mu/hm-stack/class-endpoint-controller.php

```diff
@@ -0,0 +1,238 @@
+<?php
+/**
+ * Build endpoint data for HM Stack data on this site.
+ *
+ * @package HMStackIntegration
+ */
+
+namespace HM_Stack;
+
+use WP_REST_Response;
+use WP_Error;
+use WP_Http;
+
+/**
+ * Class Endpoint_Controller
+ */
+class Endpoint_Controller extends \WP_REST_Controller {
```

Comment:
Endpoint_Controller` is a bit of a misnomer, what about HM_Stack_Controller? If it's the make controller for the namespace, `REST_Controller` would be typical.

Reply:
If there a need for a class here? I don't see any class / object state